corporate/lib/stripe.py

```diff
@@ -504,7 +504,9 @@ def make_end_of_cycle_updates_if_needed(
             standard_plan_last_ledger = (
                 LicenseLedger.objects.filter(plan=standard_plan).order_by("id").last()
             )
+            assert standard_plan_last_ledger is not None
             licenses_for_plus_plan = standard_plan_last_ledger.licenses_at_next_renewal
+            assert licenses_for_plus_plan is not None
```

Comment:
I'm not entirely sure if `licenses_for_plus_plan` will always be non-None. It will be great if anyone can confirm that.

Reply:
I think it will be ... we may want to try changing its type to `int` to enforce that, and seeing if anything complains.